tests/composition_reaction_iterated_IMPES/statistics

```diff
@@ -22,9 +22,9 @@
 # 22: Maximal value for composition C_3
 # 23: Global mass for composition C_3
 # 24: Visualization file name
-0 0.000000000000e+00 0.000000000000e+00 1024 9539 4225 12675 0  0 11  0 52 54 54 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00                                                                 "" 
+0 0.000000000000e+00 0.000000000000e+00    0    0    0     0 0  0 11  0 52 54 54 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00                                                                 "" 
```

Comment:
This doesn't look right. It says that we have zero cells, dofs of various kinds, but we took 11 iterations for composition 2, 52 iterations for the Stokes solver, etc. In other words, the information about cells and dofs doesn't seem to be correct :-(